### PR TITLE
Make event names unique.

### LINKED ISSUE #329: Events name has to be unique
One can have two events at the same location and at the same time. They can have similar details. 

The entity name should be unique for such events. Show an error if same name events are being created.

### PR DESCRIPTION
**Pull request by necessary129**
Fixes #329

### CODE REVIEW

**Comment by necessary129** [comment]
Oh well, no duplicate tests, I guess. Needs to fix those :smile:

File: vms/event/forms.py [hunk 1]
```diff
@@ -20,8 +20,9 @@ class Meta:
             ]
 
     def clean(self):
-        start_date = self.cleaned_data.get('start_date')
-        end_date = self.cleaned_data.get('end_date')
+        cleaned_data = super(EventForm, self).clean()
+        start_date = cleaned_data.get('start_date')
+        end_date = cleaned_data.get('end_date')
```
Comment (smarshy) [inline vms/event/forms.py]: @necessary129 Why this change?

Comment (necessary129) [inline vms/event/forms.py]: @smarshy I did that so that the normal validation done by django takes place. by adding the `unique` constraint and also calling the builtin validate function, the names being unique will be handled by django itself and no more code required! :D
